lib/easyify.js

```diff
@@ -14,6 +15,8 @@ module.exports = function easyify (keyFn) {
     return function makeKey (id) {
       let y
       if (isFeed(id)) {
+        if (id.startsWith('ssb:feed/ed25519')) id = toFeedSigil(id)
```

Comment:
without this line, `bfe.encode` chokes because it doesn't know the "format" `ed25519`. I'm kinda annoyed that that is what's in URI's because ed25519 is a signing curve not a feed format....  (I think I've ranted about this before. I think what I need help with is confirming a plan to unify/ align all our modules - bfe/ uri etc so they all play well together).